operator/common/images_test.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func TestDockerHubReference(t *testing.T) {
 
-	imgs := []dockerhubImage{CockroachImage, PostgresImage, FlywayImage, AlpineImage}
+	imgs := []dockerhubImage{CockroachImage, FlywayImage}
```

Comment:
ZITADELCockroachImage?

Reply:
Doesn't really matter, if you want to test all, then you have to fill in the other images as well